composer.json

```diff
@@ -3,7 +3,8 @@
     "type": "library",
     "license": "MIT",
     "require": {
-        "php": "^7.1"
+        "php": "^7.1",
+        "composer/xdebug-handler": "^1.3"
```

Comment:
This won't be necessary if we don't want to use a code coverage driver.  Happy to remove it.

Reply:
i suggest move it to require-dev section, it is not needed for production environments

Reply:
Completed.